scripts/dockerInitialize.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Define paths
+ENV_FILE="Embedded-Sharepoint/.env"
+DOCKER_COMPOSE_FILE="Embedded-Sharepoint/docker-compose.yml"
+
+# Create .env file with required environment variables
+echo "WORKDIR_MOUNT=${localWorkspaceFolder}" > $ENV_FILE
+echo "REMOTE_USER=$USER" >> $ENV_FILE
+echo "REMOTE_UID=$(id -u)" >> $ENV_FILE
+echo "REMOTE_GID=$(id -g)" >> $ENV_FILE
+
+# Bring down the docker-compose setup
```

Comment:
can you explain why this is necessary in the comment just so ppl don't look at it in the future and get confused

Reply:
partially saying this bc i don't fully remember why it has to go down